### DISCUSSION TITLE
How to change customtkinter dark mode on CTkMessagebox?

### DISCUSSION THREAD

**Discussion by ghost** [Akascape/CTkMessagebox#38] [Q&A]
How to change customtkinter dark mode on CTkMessagebox?
On ttkbootstrap or tkinter window.

**Comment by Akascape** [comment]
@informatic365 Add this line in your tkinter/ttkbootstrap app.
```python
customtkinter.set_appearance_mode("Light")
```
By default, it adapts automatically on customtkinter window.
